app/controllers/users_controller.rb

```diff
@@ -8,7 +8,11 @@ def index
     @users = User.all.page(params[:page]).per(Settings.per_page.digit_10)
   end
 
-  def show; end
+  def show
+    @microposts = @user.microposts.page(params[:page]).per(
+      Settings.per_page.digit_5
+    )
```

Comment:
xún line mới j xấu xí vậy 
```suggestion
    @microposts = @user.microposts.page(params[:page])
      .per(Settings.per_page.digit_5)
```